pyproject.toml

```diff
@@ -57,6 +56,7 @@ umap-learn = "~=0.5"
 scikit-learn = "~=1.5"
 nltk = "~=3.8"
 vocos = "~=0.1"
+huggingface-hub = {extras = ["cli"], version = "^0.24.6"}
```

Comment:
why is `cli` needed?

Reply:
not needed, planned to remove this in the final PR